spec-main/api-tray-spec.ts

```diff
@@ -12,17 +12,30 @@ describe('tray module', () => {
     tray = null as any
   })
 
-  describe('tray.setContextMenu', () => {
-    afterEach(() => {
-      tray.destroy()
+  describe('tray get/set ignoreDoubleClickEvents', () => {
+    before(function () {
+      if (process.platform !== 'darwin') this.skip()
     })
 
-    it('accepts menu instance', () => {
-      tray.setContextMenu(new Menu())
+    it('returns false by default', () => {
+      const ignored = tray.getIgnoreDoubleClickEvents()
+      expect(ignored).to.be.false
```

Comment:
this doesn't assert anything, use
```suggestion
      expect(ignored).to.be.false('ignored')
```